src/lib.rs

```diff
@@ -116,18 +120,17 @@ pub struct StreamDeck {
     kind: Kind,
     /// Connected HIDDevice
     device: HidDevice,
+    /// Image buffers updated
+    pub updated: bool,
```

Comment:
Can you instead make a function like is_updated() rather than exposing a field? I don't think it should be open to user of the library to be able to mess with this field

Reply:
I have added the is_updated() function.